mne/io/eeglab/eeglab.py

```diff
@@ -1,12 +1,15 @@
-# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.com>
 #
 # License: BSD (3-clause)
 
+import warnings
```

Comment:
Does this also go for the tests? And otherwise it's a drop-in-and-forget replacement?